cms/djangoapps/contentstore/views/course.py

```diff
@@ -570,6 +598,11 @@ def course_index(request, course_key):
         })
 
 
+def is_course_summary(course):
```

Comment:
you probably don't need to define a separate function here, especially if you take my suggestion about using a namedtuple